tests/test_rooms/test_room_filter.py

```diff
@@ -88,3 +92,17 @@ def test_filter_room_by_resources_capacity(self):
             filter_rooms_by_resources_capacity,
             filter_rooms_response
         )
+
+    def test_filter_room_by_tag(self):
+        CommonTestCases.user_token_assert_equal(
```

Comment:
Great work @Makwe-O, however, the story is asking for an admin, not a user, so the tokens being used here should be for an admin. Same goes for the other test

Reply:
Thanks for pointing this out. I will get it sorted out

Reply:
Please also add the documentation for the tests

Reply:
Thanks, I will add the documentation

Reply:
That makes sense. So there isn't any need to add the len(). I initially wanted to use the len() to check if the list was equal to zero.